flink-runtime/src/main/java/org/apache/flink/runtime/taskmanager/Task.java

```diff
@@ -27,7 +27,6 @@
 import org.apache.flink.configuration.TaskManagerOptions;
```

Comment:
Could you add a longer description to the commit message, why it's not needed anymore?

Reply:
Do you mean `FileSystemSafetyNet` usage? I can do.
My reasoning is because the thread is now only passing execution from one thread (RPC) to another (main task thread) and not calling user code anymore.

Reply:
Tbh I'm not deep enough to understand that. I just found it odd to remove a safety net that was placed for some reasons. Maybe just add your response to the commit message.

Reply:
I've added a description to the commit.